src/main/java/mate/academy/model/Book.java

```diff
@@ -24,4 +24,5 @@ public class Book {
     private String description;
     private String coverImage;
 
+

```

Comment:
Unnecessary blank line added. Consider removing it to keep the code clean and consistent.

Reply:
ok, done